### PR TITLE
seperate virsh console login function

### PR DESCRIPTION
**Pull request by nanli1**
verify_virsh_console function always auto close session
Signed-off-by: nanli <nanli@redhat.com>

For https://github.com/autotest/tp-libvirt/pull/4990  and https://github.com/autotest/tp-libvirt/pull/4963

### CODE REVIEW

**Comment by nanli1** [comment]
```
avocado run --vt-type libvirt --test-runner=runner --vt-machine-type q35  chardev.release_console

 (1/3) type_specific.io-github-autotest-libvirt.chardev.release_console.console.virtio_target: PASS (52.04 s)
 (2/3) type_specific.io-github-autotest-libvirt.chardev.release_console.serial.pci_target: PASS (46.95 s)
 (3/3) type_specific.io-github-autotest-libvirt.chardev.release_console.serial.isa_target: PASS (49.68 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

```

**Review by dzhengfy — COMMENTED** [review]
Codes LGTM. But is there any different behavior?

**Comment by nanli1** [comment]
> Codes LGTM. But is there any different behavior?

Hi dzheng, Do you mean other influences for this function?  I find it's also used in `virsh.create` so run virsh.domain job and ALL PASSED in *virsh_domain/12/ job

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -689,29 +689,7 @@ def verify_virsh_console(session, user, passwd, timeout=10, debug=False):
...
-            elif match == 2:
-                if debug:
-                    LOG.debug("Got 'Password:', sending '%s'", passwd)
-                session.sendline(passwd)
-            elif match == 3:
-                if debug:
-                    LOG.debug("Got Shell prompt -- logged in")
-                break
-
+        virsh_console_login(session, user, passwd, timeout, debug=debug)
```
Comment (chunfuwen) [inline virttest/utils_test/libvirt.py:692]: Do I miss something here? I don't notice code changes except that bringing previous code into one independent method virsh_console_login().

Comment (nanli1) [reply virttest/utils_test/libvirt.py:692]: Previous function always close console session and just return True or False. If so , We will be unable to execute some cmd with console session with previous function.

**Review by dzhengfy — APPROVED** [review]
Got it. LGTM

**Review by chunfuwen — APPROVED** [review]
lgtm

**Pull request merged**